crates/db/migrations/20230113195325_quests.sql

```diff
@@ -1,28 +1,38 @@
--- Create quests table
 CREATE TABLE IF NOT EXISTS quests (
   ID UUID PRIMARY KEY NOT NULL,
   name TEXT NOT NULL,
   description TEXT NOT NULL,
   definition bytea NOT NULL,
   created_at TIMESTAMP NOT NULL DEFAULT now(),
   updated_at TIMESTAMP NOT NULL DEFAULT now()
-  -- TODO: should we add the creator id (address)?
 );
 
--- Create quest_instances table
 CREATE TABLE IF NOT EXISTS quest_instances (
   ID UUID PRIMARY KEY NOT NULL,
-  quest_id UUID references quests(ID) ON DELETE CASCADE,  -- TODO: should not be a FK?
+  quest_id UUID references quests(ID), 
   user_address TEXT NOT NULL,
   start_timestamp TIMESTAMP NOT NULL DEFAULT now()
 );
 
--- Create events table
 CREATE TABLE IF NOT EXISTS events ( 
   ID UUID PRIMARY KEY NOT NULL,
-  quest_instance_id UUID references quest_instances(ID) ON DELETE CASCADE, -- TODO: should not be a FK?
+  quest_instance_id UUID references quest_instances(ID),
```

Comment:
I know that from business logic (or application layer) there wouldn't be any deletion but I think we could keep `ON DELETE CASCADE`. Or do you prefer to remove it for preventing?

Reply:
I would leave it without `ON DELETE CASCADE` for now as we should not be deleting things manually.